src/main/kotlin/com/openlattice/mechanic/upgrades/MaterializedEntitySetRefresh.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2019. OpenLattice, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+package com.openlattice.mechanic.upgrades
+
+import com.openlattice.mechanic.Toolbox
+import com.openlattice.postgres.*
+import org.slf4j.LoggerFactory
+
+class MaterializedEntitySetRefresh(private val toolbox: Toolbox) : Upgrade {
+    companion object {
+        private val logger = LoggerFactory.getLogger(MaterializedEntitySetRefresh::class.java)
+    }
+
+    override fun upgrade(): Boolean {
+        logger.info("Removing unused column dn_name from ${PostgresTable.ORGANIZATION_ASSEMBLIES.name} table")
+        removeDbNameColumn()
```

Comment:
probably safer to leave this column in until the migration is finished since leaving it won't break anything

Reply:
Okay, will do!